env.js

```diff
@@ -5,6 +5,10 @@ module.exports = {
   CONTENTFUL_ACCESS_TOKEN: process.env.CONTENTFUL_ACCESS_TOKEN ||
     '5c8090d12bc2bf8dc695353cc398cd5e48eb56c214325884284bfdbfef4ba5ed',
 
+  // Contentful Content Management API (only needed by admin scripts)
+  CONTENTFUL_MANAGEMENT_ACCESS_TOKEN: process.env.CONTENTFUL_MANAGEMENT_ACCESS_TOKEN ||
```

Comment:
Does it make sense to put it in the `env.js`? As far as I understand the script will run like this `node importBlogPosts.js`. So this won't be used from netlify or anywhere else?